makefile

```diff
@@ -108,6 +125,37 @@ export SHELLOPTS:=$(if $(SHELLOPTS),$(SHELLOPTS):)pipefail:errexit
 gobuildlinux = go build -ldflags="-s -w"
 gobuild = go build
 
+ifdef USE_SYSTEM_CERTS
```

Comment:
A nitpick but this doesn't need to be in an ifdef.  You could remove this out of ifdef and just put the calls to this target in the ifdef.